backend/src/geo_helpers/mod.rs

```diff
@@ -107,6 +107,37 @@ pub fn angle_of_line(line: Line) -> f64 {
     (line.dy()).atan2(line.dx()).to_degrees()
 }
 
+/// North is 0°
+/// East is 90°
+/// South  is 180°
+/// West is 270°
+pub fn euclidean_bearing(origin: Coord, destination: Coord) -> f64 {
```

Comment:
I was struggling a bit with the output range of `angle_of_line` so this simply transforms it to something I'm more familiar with. 

Is the output range of angle_of_line intentional?